RELEASING.md

```diff
@@ -34,3 +35,36 @@ Repositories create consistent release labels, such as `v1.0.0`, `v1.1.0` and `v
 The release process is standard across repositories in this org and is run by a release manager volunteering from amongst [MAINTAINERS](MAINTAINERS.md).
 
 TODO
+
+## Backporting
+
+This project follows [semantic versioning](https://semver.org/spec/v2.0.0.html). Backwards-incompatible changes always result in a new major version and will __never__ be backported. Small improvements and features will be backported to a new minor version (e.g. `1.1`). Security fixes will be backported to a new patch version (e.g. `1.0.1`).
+
+Here are the commands we typically run to backport changes to release branches:
+
+1. Checkout the target release branch and pull the latest changes from `upstream`. In the examples below, our target release branch is `1.x`.
+
+```
+git checkout 1.x
+git pull upstream 1.x
+```
+
+2. Create a local branch for the backport. A convenient naming convention is _backport-\[feature_branch_name\]-\[target-release-branch\]_.
+
+```
+git checkout -b backport-myChanges-1.x
```

Comment:
One more nit. Don't call branches `camelCase`, since those are case-sensitive. Can we replace this with `backport-issue-xyz-1.x` or `backport-pr-xyz-1.x`?